dds/DCPS/Service_Participant.h

```diff
@@ -308,6 +309,15 @@ class OpenDDS_Dcps_Export Service_Participant {
   }
 #endif
 
+#if defined(OPENDDS_NETWORK_CONFIG_UPDATER)
+
+  void add_interface(const OPENDDS_STRING &name);
+  void remove_interface(const OPENDDS_STRING &name);
+  void add_address(const OPENDDS_STRING &name, const ACE_INET_Addr& address);
+  void remove_address(const OPENDDS_STRING &name, const ACE_INET_Addr& address);
+
+#endif
+
```

Comment:
I would remove this.  A user can call network_config_monitor to get the instance and operate on it that way.